astropy/io/ascii/basic.py

```diff
@@ -287,15 +313,24 @@ def get_type_map_key(self, col):
 
 
     def get_cols(self, lines):
-        """Initialize the header Column objects from the table ``lines``.
+        """
+        Initialize the header Column objects from the table ``lines``.
 
         This is a specialized get_cols for the RDB type:
         Line 0: RDB col names
         Line 1: RDB col definitions
         Line 2+: RDB data rows
-
-        :param lines: list of table lines
-        :returns: None
+        
+        
+        Parameters
+        ----------
+        lines : list 
+            List of table lines
+        
+        Returns
```

Comment:
I think yes, because this is unusual result of execution. Also it have been mentioned in old version.